tslint.json

```diff
@@ -1,12 +1,74 @@
 {
-  "extends": "tslint:recommended",
+  "extends": ["tslint-microsoft-contrib", "tslint:recommended"],
   "rules": {
     "array-type": [true, "generic"],
     "quotemark": [true, "single", "avoid-escape"],
     "variable-name": [true, "ban-keywords"],
     "no-empty-interface": false,
     "no-var-requires": false,
     "no-string-literal": false,
-    "semicolon": [true, "ignore-interfaces"]
+    "semicolon": [true, "always"],
+    "chai-prefer-contains-to-index-of": false,
+    "chai-vague-errors": false,
+    "export-name": true,
+    "function-name": false,
+    "import-name": true,
+    "insecure-random": true,
+    "jquery-deferred-must-complete": false,
+    "max-func-body-length": false,
+    "missing-jsdoc": false,
+    "mocha-avoid-only": false,
+    "mocha-no-side-effect-code": false,
+    "mocha-unneeded-done": false,
+    "no-backbone-get-set-outside-model": false,
+    "no-banned-terms": false,
+    "no-constant-condition": false,
+    "no-control-regex": false,
+    "no-cookies": false,
+    "no-delete-expression": true,
+    "no-disable-auto-sanitization": true,
+    "no-document-domain": true,
+    "no-document-write": true,
+    "no-duplicate-case": true,
+    "no-empty-interfaces": true,
+    "no-empty-line-after-opening-brace": true,
+    "no-exec-script": true,
+    "no-for-in": true,
+    "no-function-constructor-with-string-args": true,
+    "no-function-expression": false,
+    "no-http-string": true,
+    "no-increment-decrement": false,
+    "no-inner-html": false,
+    "no-invalid-regexp": true,
+    "no-jquery-raw-elements": true,
+    "no-multiline-string": false,
+    "no-octal-literal": true,
+    "no-regex-spaces": true,
+    "no-relative-imports": false,
```

Comment:
This makes sense to be 'true'. That way we don't need to worry if we move module in folder structure.

Reply:
setting this to false will prevent `import {Comp} from 'components/Comp';` and will prefer a relative import instead `import {Comp} from '../../components/Comp';`.